app/controllers/npq_separation/admin/applications/all_controller.rb

```diff
@@ -0,0 +1,5 @@
+class NpqSeparation::Admin::Applications::AllController < NpqSeparation::AdminController
```

Comment:
Why not just `NpqSeparation::Admin::ApplicationsController`?

Reply:
I imagined there'd be several others alongside it and we might use the `NpqSeparation::Admin::ApplicationsController` for some kind of overview. All smoke and mirrors for now and likely to change.

Reply:
Flattened this to `NpqSeparation::Admin::ApplicationsController` for now, I suspect there'll be a few more flavours of applications controllers soon though.